app/models/user.rb

```diff
@@ -150,6 +150,19 @@ def self.default_challenge_manager?(email)
     /\.(gov|mil)$/.match?(email)
   end
 
+  def full_name(format: :default)
+    return "Unknown User" if first_name.blank? && last_name.blank?
```

Comment:
I'm not sure where this function but can we display the email address if the evaluator does not have any name associated? It shouldn't happen often because we require and update their first_name and last_name when they are added to the panel. `:last_first` doesn't seem to be used rn.

Reply:
I just made something since we didn't have any consistent full name functionality for users/evaluators. But I can just do what is done in the other places. Mostly wanted to make sure potential white space, missing values, or anything else was handled in the same way

Reply:
There's nothing wrong with adding the function, but I don't think it should say "Unknown User" when there is a known user, they just don't have a first_name or last_name. we have a bug in the invite flow where it's not always saving the name of the user after they accept the invite and I'd prefer not to highlight the missing name here. instead we have the email or it could use the username part of the email before the `@` symbol.

Reply:
the function is currently being used to show the name to the challenge manager and they already have access to the user's information on the evaluation panel so it's not leaking data.